ibm_db2/ibm_db2.php

```diff
@@ -49,7 +49,7 @@
  * successful. If the connection attempt fails, db2_connect
  * returns false.
  */
-function db2_connect($database, $username, $password, array $options = null) {}
+function db2_connect(string $database, ?string $username, #[\SensitiveParameter] ?string $password, array $options = []) {}
```

Comment:
Please update corresponding params in PHPDoc as well

Reply:
Done.